discovery-provider/src/queries/get_health.py

```diff
@@ -126,6 +128,12 @@ def get_health(args, use_redis_cache=True):
         latest_block_num = latest_block.number
         latest_block_hash = latest_block.hash.hex()
 
+        if use_redis_cache:
+            # if we had attempted to use redis cache and the values weren't there, set the values now
+            # ex sets expiration time and nx only sets if key doesn't exist in redis
+            redis.set(latest_block_redis_key, latest_block_num, ex=default_indexing_interval_seconds, nx=True)
+            redis.set(latest_block_hash_redis_key, latest_block_hash, ex=default_indexing_interval_seconds, nx=True)
```

Comment:
is it worth wrapping these in a try-except? wouldn't want to fail the health check due to a redis.set call failure

up to you